application/dash_application/views/pre_exome.py

```diff
@@ -283,10 +282,10 @@ def generate_mean_insert_size(current_data, colourby, shapeby, shownames,
                     )
                 ]), html.Br(),
 
-                # html.Label([
-                #     "Search Sample:",
-                #     core.Input(id=ids['search-sample'])
-                # ]), html.Br(),
+                html.Label([
+                    "Search Sample:",
+                    core.Input(id=ids['search-sample'])
+                ]), html.Br(),
```

Comment:
Looks like these got uncommented a bit early, but should be fine to merge as I know you're working on this next

Reply:
Oops! Fixed